### PR TITLE
SYS-7 convert README to asciidoc and uses third person

### PR DESCRIPTION
**Pull request by enochtsang**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+=== Setting up a Device
+
+To try running the code on a device, follow the steps found in the link:https://facebook.github.io/react-native/docs/running-on-device.html[React Native documentation for Running your app on a Device].
+
+==== Common Issues
+
+* Make sure the `JAVA_HOME` and `ANDROID_HOME` environment variables are set.
+
+* Make sure the `adb` command is available from the terminal.
+he sdk platform tools directory may have to be added to the `PATH` environment variables.
```
Comment (beaesguerra) [inline README.adoc]: I think you missed the t in the here

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+The platform tools directory can be found in `D:\Users\<your username>\AppData\Local\Android\sdk\platform-tools`.
+
+* When trying to run the application on a device, make sure a phone is connected.
+Use `adb devices` to check for a connected device.
+
+
+=== Running the Application
+
+Once a development environment has been setup, the applicatoin can be run with the following steps.
```
Comment (beaesguerra) [inline README.adoc]: Application typo here

File: README.adoc [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+. Clone the repository.
+. Run `npm install` to install the dependencies.
```
Comment (beaesguerra) [inline README.adoc]: This was my bad but could you please update the command to yarn install :)

Comment (enochtsang) [reply README.adoc]: I was wondering about that :) :+1:

**Comment by enochtsang** [comment]
Done

**Pull request merged**